### PR TITLE
refactor declarations of table and "angularCompilerOptions"

### PR DESCRIPTION
**Pull request by Mrgeniv**

### CODE REVIEW

File: src/app/components/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -12,21 +12,22 @@
...
-      <td>{{c.tank}} l</td>
-      <td>{{c.verbrauch}} l/100km</td>
-      <td>{{c.getriebe}}</td>
-      <td>{{c.ps}} PS</td>
-      <td>{{c.preis}}</td>
-      <td>{{c.sitze}}</td>
-      <td>{{c.ort}}</td>
+    <tr *ngFor="let car of cars">
+      <td>{{car.id}}</td>
+      <td><img src="{{ car.image }}" alt="" width="50"></td>
```
Comment (leku-codes) [inline src/app/components/dashboard/dashboard.component.html]: please don't use inline styles. Use a stylesheet

File: src/app/components/dashboard/dashboard.component.html [hunk 2]
```diff
@@ -12,21 +12,22 @@
       <th scope="col">Preis</th>
       <th scope="col">Sitze</th>
       <th scope="col">Ort</th>
```
Comment (leku-codes) [inline src/app/components/dashboard/dashboard.component.html]: Idea: think about more fitting titles

File: src/app/components/dashboard/dashboard.component.html (hunk 1 above)
Comment (Mrgeniv) [reply src/app/components/dashboard/dashboard.component.html]: done

File: src/app/components/dashboard/dashboard.component.html (hunk 2 above)
Comment (Mrgeniv) [reply src/app/components/dashboard/dashboard.component.html]: done

**Review by leku-codes — APPROVED** [review]

**Pull request merged**